src/main/java/com/soboleoko/medicalclinic/PatientService.java

```diff
@@ -0,0 +1,41 @@
+package com.soboleoko.medicalclinic;
+
+import com.soboleoko.medicalclinic.model.Patient;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class PatientService {
+
+    private final PatientRepository patientRepository;
+
+    @Autowired
```

Comment:
better to use @RequiredArgsConstructor becouse we have only one constructor

Reply:
użyj Lomboka